src/user-guide/writing-workflows/external-triggers.rst

```diff
@@ -53,22 +105,24 @@ External triggers are configured in the
 Built-in Clock Triggers
 -----------------------
 
-These are more transparent (exposed in the graph) and efficient (shared among
-dependent tasks) than the older clock triggers described
-in :ref:`ClockTriggerTasks`.
+Clock xtriggers succeed when the real ("wall clock") time reaches some offset
+from the task's cycle point value.
 
-Clock triggers, unlike other trigger functions, are executed synchronously in
-the main process. The clock trigger function signature looks like this:
+.. note::
+
+   These should be used instead of the older task clock triggers documented in
+   :ref:`ClockTriggerTasks`.
+
+The clock xtrigger function signature looks like this:
 
 .. autofunction:: cylc.flow.xtriggers.wall_clock.wall_clock
 
-The ``offset`` argument is a datetime duration (``PT1H`` is 1
-hour) relative to the dependent task's cycle point (automatically passed to the
-function via a second argument not shown above).
+The ``offset`` argument is a datetime duration (e.g., ``PT1H`` is 1 hour)
```

Comment:
Does this add anything to content of the function doc? I'm not sure it's needed?

Reply:
I'll remove it.